sources/engine/Stride.Games/Stride.Games.csproj

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <StrideRuntime>true</StrideRuntime>
-    <StrideRuntimeWindowsNet6>true</StrideRuntimeWindowsNet6>
```

Comment:
This variable should have a more explicit name, from what I understand this is used to force dotnet to produce an assembly specifically for windows ... ? Probably to affect preprocessors in game and input ? So, something like `StrideExplicitWindowsRuntime` maybe ?

Reply:
I think your suggestion about the variable name captures its meaning well 👍. Lets do that